app/src/main/res/layout-large-land/player.xml

```diff
@@ -345,15 +345,58 @@
                         android:padding="@dimen/player_main_buttons_padding"
                         android:scaleType="fitCenter"
                         android:visibility="gone"
-                        app:tint="@color/white"
                         app:srcCompat="@drawable/ic_fullscreen"
+                        app:tint="@color/white"
                         tools:ignore="ContentDescription,RtlHardcoded"
                         tools:visibility="visible" />
 
                 </LinearLayout>
 
             </LinearLayout>
 
+            <LinearLayout
```

Comment:
Is there a way to prevent nesting layouts? I guess not, seeing how you use the bottom layout in Player.java, so don't worry

Reply:
Sry for the late reply, didn't see the changes because there were other comments posted (below) in nearly the same time.

I first tried to do it without nested layouts however then I had massive problems with that code:
https://github.com/TeamNewPipe/NewPipe/pull/6434/files#diff-68cd2f051cf461f1cee8f97ab4a793ab90276943eed26f8f1baf5105948b2b5eR1724-R1729
(see Stackoverflow for explanation)

So I simply added a new layout and the problems were gone.
The new layout is also a kind of placeholder, where the preview can occur 😄 

If someone knows a better way to solve that, please enlighten me 😉

Reply:
Ok, no problem then